scripts/migration/migrate_personal_to_profile_websites.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Script to migrate nodes with invalid categories."""
+
+import logging
+
+from modularodm import Q
+from nose.tools import *
+from tests.base import OsfTestCase
+
+from website import models
+from website.app import init_app
+from scripts import utils as scripts_utils
+from tests.factories import AuthUserFactory
+
+
+logger = logging.getLogger(__name__)
+
+
+def main():
+    init_app(routes=False)
+    dry_run = 'dry' in sys.argv
+    if not dry_run:
+        scripts_utils.add_file_logger(logger, __file__)
+    logger.info("migrating personal to profileWebsites")
+            
+    for user in get_users_with_personal_websites():
+        logger.info(repr(user))
+        logger.info(repr(user.social))
+        if not user.social.get('profileWebsites', None):
+            user.social['profileWebsites'] = []
+            if user.social.get('personal'):
+                migrate_personal_to_profile_websites(user)
+        logger.info(repr(user.social))
+        if not dry_run:
+            user.save()
+
+
+def get_users_with_personal_websites():
```

Comment:
Since this is getting users with a non-empty social field, maybe rename this to `get_users_with_social_field`